samples/automl/automlVisionModel.js

```diff
@@ -0,0 +1,607 @@
+
+  // Get the latest state of a long-running operation.
+
+  client.operationsClient.getOperation(operationFullId).then(responses => {
+    const response = responses[0];
+    console.log(`Operation status: `, response);
+  });
+  // [END automl_vision_getOperationStatus]
+}
+
+function listModels(projectId, computeRegion, filter_) {
+  // [START automl_vision_listModels]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const filter_ = `filter expressions, must specify field, e.g. “imageClassificationModelMetadata:*”`;
+
+  // A resource that represents Google Cloud Platform location.
+  const projectLocation = client.locationPath(projectId, computeRegion);
+
+  // List all the models available in the region by applying filter.
+  client
+    .listModels({parent: projectLocation, filter: filter_})
+    .then(responses => {
+      const model = responses[0];
+
+      // Display the model information.
+      console.log(`List of models:`);
+      for (let i = 0; i < model.length; i += 1) {
+        console.log(`Model name: `, model[i].name);
+        console.log(`Model id: `, model[i].name.split(`/`).pop(-1));
+        console.log(`Model display name: `, model[i].displayName);
+        console.log(`Model dataset id: `, model[i].datasetId);
+        if (model[i].modelMetadata === `translationModelMetadata`) {
+          console.log(`Translation model metadata:`);
+          console.log(
+            `\tBase model: `,
+            model[i].translationModelMetadata.baseModel
+          );
+          console.log(
+            `\tSource language code: `,
+            model[i].translationModelMetadata.sourceLanguageCode
+          );
+          console.log(
+            `\tTarget language code: `,
+            model[i].translationModelMetadata.targetLanguageCode
+          );
+        } else if (
+          model[i].modelMetadata === `textClassificationModelMetadata`
+        ) {
+          console.log(
+            `Text classification model metadata: `,
+            model[i].textClassificationModelMetadata
+          );
+        } else if (
+          model[i].modelMetadata === `imageClassificationModelMetadata`
+        ) {
+          console.log(`Image classification model metadata:`);
+          console.log(
+            `\tBase model id: `,
+            model[i].imageClassificationModelMetadata.baseModelId
+          );
+          console.log(
+            `\tTrain budget: `,
+            model[i].imageClassificationModelMetadata.trainBudget
+          );
+          console.log(
+            `\tTrain cost: `,
+            model[i].imageClassificationModelMetadata.trainCost
+          );
+          console.log(
+            `\tStop reason: `,

```

Comment:
As mentioned in the other PR - can we move each of these `console.log` blocks into its own helper function?

Apply this comment throughout your PR.

Reply:
Also - `model[i].imageClassificationModelMetadata` should probably be in a variable (so we don't have so many two-line `console.log` statements).